src/buffer/out/textBuffer.cpp

```diff
@@ -2445,16 +2493,18 @@ void TextBuffer::_AppendRTFText(std::ostringstream& contentBuilder, const std::w
             case L'\\':
             case L'{':
             case L'}':
-                contentBuilder << "\\" << gsl::narrow<char>(codeUnit);
+                fmt::format_to(std::back_inserter(contentBuilder), FMT_COMPILE("\\{}"), gsl::narrow_cast<char>(codeUnit));

```

Comment:
i love all the formatting, but this one is probably just simpler as `contentBuilder += "\\"` and then falling through to the `default` case with an explicit `[[fallthrough]]`